packages/components/rollup.config.js

```diff
@@ -13,7 +13,19 @@ export default {
     }),
     postcss({
       modules: true,
-      plugins: [require("postcss-import"), require("autoprefixer")],
+      plugins: [
+        require("postcss-import"),
+        require("autoprefixer"),
```

Comment:
if my assumptions are correct, `postcss-preset-env` has autoprefixers too... which raises the question, do we need this? Or should we just remove `postcss-preset-env` and just install the things we need